### PR TITLE
Indicate consistent switchover problems in WebUI

### PR DESCRIPTION
**Pull request by rosik**
What has been done? Why? What problem is being solved?

I didn't forget about

- [x] Tests
- [x] Changelog (unnecessary)
- [x] Documentation

Part of #908

### CODE REVIEW

File: cartridge/lua-api/failover.lua [hunk 1]
```diff
@@ -170,27 +171,30 @@ local function promote(replicaset_leaders, opts)
         return nil, PromoteLeaderError:new('There is no active coordinator')
     end
 
-    local _, err = rpc.call(
+    local ok, err = rpc.call(
```
Comment (olegrok) [inline cartridge/lua-api/failover.lua:174]: Don't understand why this change is necessary

File: cartridge/lua-api/failover.lua [hunk 2]
```diff
@@ -170,27 +171,30 @@ local function promote(replicaset_leaders, opts)
...
     end
 
-    if opts == nil or opts.force_inconsistency == false then
-        return true
+    if opts ~= nil and opts.force_inconsistency == true then
+        local ok, err = failover.force_inconsistency(replicaset_leaders)
+        if ok == nil then
+            return nil, PromoteLeaderError:new(
+                "Promotion succeeded, but inconsistency wasn't forced: %s",
+                errors.is_error_object(err) and err.err or err
```
Comment (olegrok) [inline cartridge/lua-api/failover.lua:190]: Why isn't it simply `tostring(err)`. Seems it should work in the same way for simple strings and errors from "errors" module

Comment (rosik) [reply cartridge/lua-api/failover.lua:190]: tostring(err) includes stack traceback. Since I create new error object, i don't want it to become a part of new error message.

File: cartridge/lua-api/failover.lua (hunk 1 above)
Comment (rosik) [reply cartridge/lua-api/failover.lua:174]: It isn't necessary. I can remove it if you insist.

Comment (olegrok) [reply cartridge/lua-api/failover.lua:174]: It depends on the way how errors are handled in cartridge. I'd prefer to use `err ~= nil` but if `ok == nil` is preferable for the cartridge, then you should leave this as is.

Comment (rosik) [reply cartridge/lua-api/failover.lua:174]: I think checking `ret == nil` is more suitable in general. The only exception are methods that can return `nil` without an error, but it's not the case

Comment (rosik) [reply cartridge/lua-api/failover.lua:174]: So I'll leave it as is.
